swagger.yaml

```diff
@@ -5479,6 +5480,23 @@ definitions:
       initrd_url:
         type: string
         description: URL address to download the initrd.
+      host_fs_mount_dir:
```

Comment:
How does the service know this?  Can the agent detect it so that it's more future-proof?

Reply:
Currently it's hard-coded [here](https://github.com/openshift/assisted-service/blob/347339cb7e089c34d51e7085d9fe20c16826b319/internal/controller/controllers/agent_reclaimer.go#L159), but I've made it an assisted-service environment variable in my latest [PR](https://github.com/openshift/assisted-service/pull/4271/files#diff-5fb1b62b486cd67165e89476a57536baa3fc9822734d1e2922b8a9c9484e9f18R32). May I get feedback on that? Should it be provided in a different way?

Good question about agent detection, since the agent is running in a pod, it could also be provided as an environment variable of the pod itself when the agent is spun up. @carbonin what do you think?

Reply:
Either way we should have one place for the source of this data.

I suppose we could provide it as an env var rather than through the request, but I recall thinking that adding more env vars or args to the agent that only applied to a single operation didn't seem like a good pattern.

> Can the agent detect it so that it's more future-proof?

I think @avishayt is asking if there's something we can look at in the `mount` output that might tell us where the host rootfs is mounted.

Maybe, but I'm not sure it would be particular straight forward.